payments-core/src/main/java/com/stripe/android/view/AuthActivityStarterHost.kt

```diff
@@ -19,7 +19,7 @@ sealed class AuthActivityStarterHost {
         requestCode: Int
     )
 
-    internal abstract val statusBarColor: Int?
+    abstract val statusBarColor: Int?
```

Comment:
Is this needed?

Reply:
Yes, as we want to pass the calling activity’s status bar color over. We also do this in various other places (for instance `PaymentSheetActivity`).